### ISSUE TITLE
Learning rate assignment

### ISSUE THREAD

**Issue by choice17** [ultralytics/yolov5#981]
## ❔Question

Why are the batchnorm weights(pq0) being assigned with hyp['lr0'] but not for all convolution weights(pq1)?
Is it designed for purpose?
Isn't it implies that learning rate for convolution weights are using pytorch default setting?

https://github.com/ultralytics/yolov5/blob/master/train.py#L95-L118

```python
    pg0, pg1, pg2 = [], [], []  # optimizer parameter groups
    for k, v in model.named_parameters():
        v.requires_grad = True
        if '.bias' in k:
            pg2.append(v)  # biases
        elif '.weight' in k and '.bn' not in k:
            pg1.append(v)  # apply weight decay convolution weight in pg1
        else:
            pg0.append(v)  # all else batchnorm ... etc

if opt.adam:
        optimizer = optim.Adam(pg0, lr=hyp['lr0'], betas=(hyp['momentum'], 0.999))  # adjust beta1 to momentum
else:
        optimizer = optim.SGD(pg0, lr=hyp['lr0'], momentum=hyp['momentum'], nesterov=True) # just for batchnorm weight only

optimizer.add_param_group({'params': pg1, 'weight_decay': hyp['weight_decay']})  # add pg1 with weight_decay
optimizer.add_param_group({'params': pg2})  # add pg2 (biases)
```

The cosine learning rate scheduler is using hyp['lrf'] for all learning rate in optimizer.

**Comment by choice17** [comment]
I think optimizer takes lr in constructor, and assign lr to all parameters group as default.
```python
optimizer = optim.SGD(pg0, lr=hyp['lr0'], momentum=hyp['momentum'], nesterov=True) # just for batchnorm weight only
```

**Comment by feiyuhuahuo** [comment]
Same question, have you figured out why? @choice17
